docs/documentation/style-guide.md

```diff
@@ -28,16 +28,24 @@ Use the following conventions for headings:
 Links
 -----
 
-!!! danger "Avoid document relative links"
-    Document-relative links to other pages have changed in MkDocs 1.0.0 in a backwards-incompatible way.
-    Please convert any document-relative links to site-relative links before updating the `doc-ci-scripts` submodule.
+!!! danger "Only use document relative links in MkDocs 1.0.0 and newer"
+    MkDocs 1.0.0 does not support site-relative links (e.g. `/software/development-process`).
+    You must use document-relative links (e.g. `../software/development-process`) instead.
+
+    Earlier versions of this guide recommended site-relative links;
+    these only worked in earlier versions of MkDocs.
+
+    Please convert any site-relative links to document-relative links
+    before updating the `doc-ci-scripts` submodule to use MkDocs 1.0.0+.
 
 - Links to internal pages should not have the `.md` extension
```

Comment:
```suggestion
```

The MkDocs documentation uses the `.md` suffix so let's go with that. Though I think it's fine either way

Reply:
The JIRA ticket says the opposite:
> Any internal links should not have the .md suffix

Reply:
Argh, sorry. That was an oversight on my part. Further down in the description I added:

> Fix internal technology links (they need to be doc-relative with .md suffixes according to https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

Reply:
Wait, do we _need_ to add the .md suffixes everywhere? Turning `](/` into `](../` was pretty easy, but the other one is going to be a pretty grody regex to write...

Reply:
I *think* it works without the suffix so I'm open to excluding that requirement

Reply:
Tested it and no, it really doesn't. I modified the text to add the md requirement.